db/R/get.trait.data.R

```diff
@@ -142,6 +141,18 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
           for(id in 1:nrow(files)) {
             file.copy(file.path(files[[id, 'file_path']], files[[id, 'file_name']]), file.path(pft$outdir, files[[id, 'file_name']]))
           }
+          
+          # May need to symlink the generic post.distns.Rdata to a specific post.distns.*.Rdata file.
```

Comment:
This block is unrelated (sorry) but makes meta-analysis update=AUTO work as expected.